spaces/S000031/properties/P000204.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000031
+property: P000204
+value: false
+refs:
+- mathse: 301095
+  name: A Product of Connected Spaces Minus A Product of Proper Subsets of Both
+---
+
+Since {S29|P36}, from {{mathse:301095}} it follows that if you remove a point from $X$ then it is connected. 
```

Comment:
```suggestion
- mathse: 214317
  name: $(X\times Y)\setminus (A\times B)$ is connected for $A,B$ proper subsets of connected $X,Y$
---

Since {S29|P36}, from {{mathse:214317}} it follows that if you remove a point from $X$ then it remains connected.
```